draft-ietf-acme-acme.md

```diff
@@ -1007,8 +1007,7 @@ caching of this resource.
 
 A client creates a new account with the server by sending a POST request to the
 server's new-account URL.  The body of the request is a stub account object
-containing the "contact" field and optionally the "termsOfServiceAgreed"
-field.
+optionally containing "contact" and the "termsOfServiceAgreed" fields.
```

Comment:
small nit: I think this reads clearer as "optionally containing **the** "contact" and "termsOfServiceAgreed" fields."

Reply:
Thanks for correction, @cpu - fixed it